build/script/steamcmd

```diff
@@ -1,7 +1,3 @@
 #!/usr/bin/env bash
 
-if [ `whoami` != "$STEAM_USER" ]; then
-	exec gosu $STEAM_USER $0 "$@"
-fi
-
```

Comment:
I'm ok with the other changes of this pr but I don't like this one. This `if` is handy and it prevent human errors.